woocommerce/woocommerce-android
# [Customer Search] Edit customer flow

Closes: #9405


### Description

The PR modifies the "edit customers details" flow. Now it possible to delete customer details

### Testing instructions

* Start order creation
* Add customer details
* Try to edit the customer
* Notice that it can be edited or deleted

**Important** to test that on release build and make sure that non-feature flagged flow works as before the change

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/5113450c-6d27-43ee-bd24-ff50f606994c
